docs/source/transforms.rst

```diff
@@ -11,732 +11,776 @@ Generic Interfaces
 .. currentmodule:: monai.transforms
 
 `Transform`
-~~~~~~~~~~~
+^^^^^^^^^^^
 .. autoclass:: Transform
     :members:
     :special-members: __call__
 
 `MapTransform`
-~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^
 .. autoclass:: MapTransform
     :members:
     :special-members: __call__
 
 `Randomizable`
-~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^
 .. autoclass:: Randomizable
     :members:
 
 `Compose`
-~~~~~~~~~
+^^^^^^^^^
 .. autoclass:: Compose
     :members:
     :special-members: __call__
 
-
 Vanilla Transforms
 ------------------
 
-`Spacing`
-~~~~~~~~~
-.. autoclass:: Spacing
-    :members:
-    :special-members: __call__
-
-`Orientation`
-~~~~~~~~~~~~~
-.. autoclass:: Orientation
-    :members:
-    :special-members: __call__
+Crop and Pad
```

Comment:
To align with the transform source code structure, should we use "Crop and Pad (Array)"?
I think maybe it's more clear for users to search expected transform and align with the "Crop and Pad (Dict)".

Thanks.

Reply:
I actually didn't want to put "(Dict)". but I got some sphinx warnings about duplicated section titles...